napari_animation/animation.py

```diff
@@ -187,15 +188,15 @@ def animate(
                 ]:
                     writer = imageio.get_writer(
                         filename,
-                        fps=fps,
+                        duration=duration,
```

Comment:
```suggestion
                        fps=fps,
```
Only the pillow plugin is using duration